src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react';
+import { Container } from '@material-ui/core';
+import { StrengthsWeaknessesForm } from './StrengthsWeaknessesForm';
```

Comment:
add index.ts in src/pages/strengths-weaknesses-page which exports at least StrengthsWeaknessesPage